client/termsetting/handlers/numeric.discrete.ts

```diff
@@ -511,44 +494,25 @@ function renderLastBinInputs(self: NumericTermSettingInstance, tr: any) {
 		.attr('type', 'number')
 		.property('value', self.q.last_bin ? self.q.last_bin.start : '')
 		.style('width', '100px')
-		.style('margin-left', '15px')
-		.on('change', handleChange)
-		.on('keyup', function (this: any, event: any) {
-			if (!keyupEnter(event)) return
-			handleChange.call(this)
-		})
+		.on('change', setLastBinStart)
 
-	// note div
-	tr.append('td')
-		.style('display', 'none')
+	edit_div
 		.append('div')
 		.style('font-size', '.6em')
-		.style('margin-left', '1px')
-		.style('padding-top', '30px')
-		.style('color', '#858585')
+		.style('opacity', 0.5)
 		.style('display', self.num_obj.no_density_data ? 'none' : 'block')
-		.html('<b>Right</b>most red line indicates the last bin start. <br> Drag that line to edit this value.')
-
-	function handleChange() {
-		self.q.last_bin!.start = +self.dom.last_start_input.property('value')
-		self.dom.last_start_input.restyle()
-		self.renderBinLines(self, self.q as NumericQ)
-		if (self.dom.last_radio_auto.property('checked')) {
-			delete self.q.last_bin?.start
-			edit_div.style('display', 'none')
+		.text('Indicated by right-most red line. Drag to change.')
```

Comment:
'Indicated by the right-most red line. Drag this line to change the bins' ??

Reply:
i mean "drag to change the value in the input box". hope that's clear enough